bin/codeclimate-test-reporter

```diff
@@ -1,24 +1,36 @@
 #!/usr/bin/env ruby
 
 require "codeclimate-test-reporter"
+require "optparse"
 
-COVERAGE_FILE = "coverage/.resultset.json".freeze
```

Comment:
Sure, no problem. Using option parser is maybe premature at this point. I'll update the pr.